### PR TITLE
Merge Request for Error Handling JAVA8MIG-241

### PR DESCRIPTION
**Pull request by chhex**
I have adopted the Server and Apscli the the new ExceptionHandling resp. Errorhandling. 
Î will not merge before tomorrow (Donnerstag, 7.6 ) 15:30.  
Also i will Install local the rpm 's locally and test as far as possible. 

Note: it is well possible that not all Corner Cases in terms of Exceptions are handled well yet. Please Feedback. 

I suggest that i maybe do the adoptation of the Java Client (JavaForms) to the new Error Handling i will do in a second step, after merging.  Depending on the timeline

### CODE REVIEW

File: apg-patch-service-cmdclient/src/main/groovy/com/apgsga/patch/service/client/PatchCli.groovy [hunk 1]
```diff
@@ -41,85 +42,103 @@ class PatchCli {
...
+			cmdResults.results['error'] = e.message
+			return cmdResults
+
+		} catch (Exception e) {
+			System.err.println " Unhandling Exception occurred "
+			System.err.println e.toString()
+			StackTraceUtils.printSanitizedStackTrace(e,new PrintWriter(System.err))
+			cmdResults.results['error'] = e
+			return cmdResults
 		}
```
Comment (apgsga-jhe) [inline apg-patch-service-cmdclient/src/main/groovy/com/apgsga/patch/service/client/PatchCli.groovy]: Cool, this new try/catch will be used as well for https://jira.apgsga.ch/browse/JAVA8MIG-365 as soon as this pull request will be merged.

**Comment by chhex** [comment]
@apgsga-jhe I can do the merge  asap , meaning in the next hour, which would be convient for me. I just thought, that we wait until the feedback workshop is through?  But then again, we do not have to install until after 15:00?

**Comment by chhex** [comment]
@apgsga-jhe ok good: i'll merge and not install.

**Comment by chhex** [comment]
@apgsga-jhe i just merged.  Build and tests look are ok. Maybe if you can give it a quick check, if all your code is still there ;-)
